src/index.js

```diff
@@ -1,14 +1,23 @@
-import _ from 'lodash';
+// import _ from 'lodash';
```

Comment:
* Kindly remove commented code from your codebase, as it makes your code untidy 🙏